tests/core/data/test_parquet_data_node.py

```diff
@@ -300,6 +300,50 @@ def test_get_system_folder_modified_date_instead_of_last_edit_date(self, tmpdir_
 
         os.unlink(temp_file_path)
 
+    @pytest.mark.parametrize(
+        "content",
+        [
+            ([{"a": 11, "b": 22, "c": 33}, {"a": 44, "b": 55, "c": 66}]),
+            (pd.DataFrame([{"a": 11, "b": 22, "c": 33}, {"a": 44, "b": 55, "c": 66}])),
+        ],
+    )
+    def test_append_pandas(self, parquet_file_path, default_data_frame, content):
+        # !!! append data only works with `fastparquet`
+        if not util.find_spec("fastparquet"):
+            pass
```

Comment:
Maybe we can raise a proper error when using append without fastparquet, to make it clean.

Reply:
Oh this is for the test. If someone install taipy-core and do not install the optional fastparquet library, the useless error will not be raised

Reply:
And what if a user install taipy-core with fastparquet? What is the behavior when calling append?

Reply:
Then whatever the engine defined at the config level is, the append method will use fastparquet.

This is to avoid the situation when the user wants to use pyarrow for a specific reason, but still want to append data, then we do allow that.